packages/mint-components/src/components/sqm-navigation-sidebar-item/sqm-navigation-sidebar-item-view.tsx

```diff
@@ -47,7 +48,11 @@ export function NavigationSidebarItemView(
   const styleString = sheet.toString();
 
   return (
-    <div onClick={callbacks.onClick} class={sheet.classes.ItemContainer}>
+    <div
+      onClick={callbacks.onClick}
+      onMouseDown={(e) => (e.button === 1 ? callbacks.onMiddleClick(e) : true)}
+      class={sheet.classes.ItemContainer}
+    >
```

Comment:
These should just be simple `<a href="/wherever">` components, without all the special `onMouseDown` handling.